### PR TITLE
Fix jsdoc building

### PR DESCRIPTION
**Pull request by tmcw**

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -85,89 +85,115 @@ fs.writeFileSync('./style.json', JSON.stringify(migrate(style)));
...
+
+### Parameters
+
+| parameter | type | description               |
+| --------- | ---- | ------------------------- |
+| `a`       | alu  | potentially-bundled value |
+
+
 
+**Returns** `Object`, an unbundled value
```
Comment (lucaswoj) [inline API.md]: This function :point_up_2: should be private

File: API.md [hunk 2]
```diff
@@ -85,89 +85,115 @@ fs.writeFileSync('./style.json', JSON.stringify(migrate(style)));
...
-Validate a Mapbox GL Style given a JSON object against the latest
-version of the style spec. Returns results in the same format as
-`validate`.
+### `exports`
+
+Validate a style against the latest specification. This method is optimized
+to keep its bundle size small by refraining from requiring jslint or old
+style spec versions.
+
+
```
Comment (lucaswoj) [inline API.md]: This method :point_up_2: is deprecated and should not appear in the docs

File: API.md [hunk 3]
```diff
@@ -85,89 +85,115 @@ fs.writeFileSync('./style.json', JSON.stringify(migrate(style)));
...
+| parameter       | type               | description  |
+| --------------- | ------------------ | ------------ |
+| ``              | ption              |              |
+| `[options.key]` | string             | _optional:_  |
+| ``              | ptions\.valu       |              |
+| ``              | options\.valueSpec |              |
+| ``              | options\.style     |              |
+| ``              | options\.styleSpec |              |
+| ``              | options\.layer     |              |
+| ``              | ptions\.objectKe   |              |
```
Comment (lucaswoj) [inline API.md]: This method :point_up_2: is improperly named. It looks like one of the `validate*` methods (i.e. `validateLayer`, `validateLayoutProperty`)

File: API.md [hunk 4]
```diff
@@ -85,89 +85,115 @@ fs.writeFileSync('./style.json', JSON.stringify(migrate(style)));
...
 
-**Returns** `Array.<Object>`, an array of errors
```
Comment (lucaswoj) [inline API.md:171]: This method :point_up_2: is improperly named. It looks like one of the `validate*` methods (i.e. `validateLayer`, `validateLayoutProperty`)

**Comment by tmcw** [comment]
@lucaswoj this only changes one line of source, to fix a syntax error in jsdoc https://github.com/mapbox/mapbox-gl-style-spec/pull/432/files#diff-b39ac4af8c3f9bf4b488429e5a32e17cL6 - should i not generate docs for now, since there are problems this brought to the surface?

**Comment by lucaswoj** [comment]
Ah! I see. We should fix the problems in this PR before merging. Do you mind tackling them or shall I?

**Comment by tmcw** [comment]
Go for it, I'm just fixing an issue that's been blocking publisher - i'm not hip to it ™ wrt the current codebase

File: lib/validate/latest.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-/**
+/*
```
Comment (lucaswoj) [inline lib/validate/latest.js:3]: not the best way to hide documentation but I don't really feel like wrangling dox today

**Comment by lucaswoj** [comment]
:ship:

**Pull request merged**